graphbolt/src/cnumpy.h

```diff
@@ -0,0 +1,65 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file cnumpy.h
+ * @brief Numpy File Fetecher class.
+ */
+
+#ifdef __linux__
+#include <stdint.h>
+#include <torch/script.h>
+
+#include <cassert>
+#include <cstdio>
+#include <cstring>
+#include <iostream>
+#include <string>
+
+namespace graphbolt {
+namespace storage {
+
+/**
+ * @brief Disk Numpy Fetecher class.
+ */
+class OnDiskNpyArray {
```

Comment:
Let's expose this class to torch so that it can be returned to python. Let's also make sure to open the file once and keep it open until destruction.

Reply:
Hi guy, I have expose this class and change to use torch parallel, maybe you can take a look at it. :)